tasks/suse-install.yml

```diff
@@ -1,10 +1,5 @@
 ---
 # setup repo, using official SUSE NHC repo
-- name: Ensure NHC OBS repo is setup
```

Comment:
I thought the warewulf-nhc package is only found in the network-cluster repo, is it now in SLE repos?

Reply:
no, but we add the network-cluster repo!